### PR TITLE
FIX] web: infinity error loop when errors on multiple actions in the URL

### PR DESCRIPTION
**Pull request by jpp-odoo**
- Open Accounting;
- Open Bank Reconciliation;
- Change the Company, to a company with not installed a chart of account.

Before this commit, an infinity access error loop was raise. This occurs,
because when an action is in an error on mounting on the action service,
we try to mount the previous action, choose from a list of previously
executed actions. The previous action, in this case, was also in error.
In these case, it should go to the before last of the list, but as the
previous one wasn't removed. It will try to mount it again, and create
an infinity loop.

opw-[4076702](https://www.odoo.com/web#id=4076702&view_type=form&model=project.task)

### CODE REVIEW

File: addons/web/static/src/webclient/actions/action_service.js [hunk 1]
```diff
@@ -1498,7 +1498,7 @@ export function makeActionManager(env, router = _router) {
                 throw new Error("Attempted to restore a virtual controller whose state is invalid");
             }
             const { actionRequest, options } = actionParams;
-            options.index = index;
+            controllerStack = controllerStack.slice(0, index);
```
Comment (kebeclibre) [inline addons/web/static/src/webclient/actions/action_service.js:1501]: `delete options.index` ?

**Comment by kebeclibre** [comment]
robodoo delegate+

**Comment by kebeclibre** [comment]
robodoo rebase-ff

**Comment by jpp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**